NPM badges fail when dependencies are not semver

```
  3) NodeVersion
       gets the node version of @stdlib/stdlib
         
	[ GET /@stdlib/stdlib.json ]:
     AssertionError: expected [Function] to not throw an error but 'TypeError: Invalid comparator: invalid' was thrown
      at expectSemverRange (services/node/node.tester.js:10:46)
      at IcedFrisbyNock.t.create.get.expectJSONTypes.afterJSON.json (services/node/node.tester.js:24:5)
      at IcedFrisbyNock.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:902:10)
      at invokeNextHook (node_modules/icedfrisby/lib/icedfrisby.js:1016:22)
      at IcedFrisbyNock._runHooks (node_modules/icedfrisby/lib/icedfrisby.js:1029:5)
      at IcedFrisbyNock._finish (node_modules/icedfrisby/lib/icedfrisby.js:1308:12)
      at IcedFrisbyNock._invokeExpects (node_modules/icedfrisby/lib/icedfrisby.js:1270:10)
      at start (node_modules/icedfrisby/lib/icedfrisby.js:1244:12)
      at Request.runCallback [as _callback] (node_modules/icedfrisby/lib/icedfrisby.js:1131:16)
      at Request.self.callback (node_modules/request/request.js:185:22)
      at Request.<anonymous> (node_modules/request/request.js:1161:10)
      at IncomingMessage.<anonymous> (node_modules/request/request.js:1083:12)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:139:11)
      at process._tickDomainCallback (internal/process/next_tick.js:219:9)
```

https://circleci.com/gh/badges/daily-tests/33

--- a/services/npm/npm-base.js
+++ b/services/npm/npm-base.js
@@ -10,7 +10,15 @@ const deprecatedLicenseObjectSchema = Joi.object({
   type: Joi.string().required(),
 })
 const dependencyMap = Joi.object()
-  .pattern(/./, semverRange)
+  .pattern(
+    /./,
+    Joi.alternatives().try(
+      semverRange,
+      Joi.string()
+        .uri()
+        .required()
+    )
+  )
   .default({})
 const schema = Joi.object({
   dependencies: dependencyMap,

